targets/esp8266/user/jerry_run.c

```diff
@@ -94,23 +68,25 @@ int js_loop (uint32_t ticknow)
     return -2;
   }
 
-  val_argv = 1;
-  val_args[0] = jerry_create_number (ticknow);
 
-  res = jerry_call_function (sysloop_func,
+  jerry_value_t val_args[] = { jerry_create_number (ticknow) };
+  uint16_t val_argv = sizeof (val_args) / sizeof (jerry_value_t);
+
+  jerry_value_t res = jerry_call_function (sysloop_func,
                              global_obj_val,
                              val_args,
                              val_argv);
 
   if (jerry_value_has_error_flag (res)) {
-    ret_code = -3;
+    jerry_release_value (sysloop_func);
+    jerry_release_value (global_obj_val);
+    return -3;
   }
 
-  jerry_release_value (res);
```

Comment:
`res` is not released anymore (neither here, not in the error handling branch above)